stdlib/public/Platform/POSIXError.swift

```diff
@@ -682,4 +682,202 @@ public enum POSIXErrorCode : Int32 {
     case STRUNCATE      = 80
 }
 
+#elseif os(OpenBSD)
+
+/// Enumeration describing POSIX error codes.
+public enum POSIXErrorCode : Int32 {
+    /// Operation not permitted
+    case EPERM			= 1
+    /// No such file or directory
+    case ENOENT			= 2
+    /// No such process
+    case ESRCH			= 3
+    /// Interrupted system call
+    case EINTR			= 4
+    /// Input/output error
+    case EIO			= 5
```

Comment:
What's with the inconsistent formatting?

Reply:
Tabs align the equal signs but they don't seem to be rendering here. Should I use spaces?

Reply:
Don't care.